science/SDRangel/Portfile

```diff
@@ -0,0 +1,150 @@
+    -DRX_SAMPLE_24BIT=ON \
+    -DFORCE_SSE41=ON \
+    -DBUILD_GUI=OFF \
+    -DBUILD_SERVER=OFF \
+    -DENABLE_SOAPYSDR=OFF \
+    -DENABLE_AIRSPY=OFF \
+    -DENABLE_AIRSPYHF=OFF \
+    -DENABLE_BLADERF=OFF \
+    -DENABLE_FUNCUBE=OFF \
+    -DENABLE_HACKRF=OFF \
+    -DENABLE_IIO=OFF \
+    -DENABLE_LIMESUITE=OFF \
+    -DENABLE_MIRISDR=OFF \
+    -DENABLE_PERSEUS=OFF \
+    -DENABLE_RTLSDR=OFF \
+    -DENABLE_XTRX=OFF \
+    -DBUNDLE=OFF
+
+variant soapysdr description {Enable SoapySDR support} {
+    depends_lib-append      port:SoapySDR
+    configure.args-replace  -DENABLE_SOAPYSDR=OFF -DENABLE_SOAPYSDR=ON
+}
+
+variant airspy description {Enable Airspy hardware} {
+    depends_lib-append      port:airspy
+    configure.args-replace  -DENABLE_AIRSPY=OFF -DENABLE_AIRSPY=ON
+}
+
+variant airspyhf description {Enable AirspyHF hardware} {
+    depends_lib-append      port:airspyhf
+    configure.args-replace  -DENABLE_AIRSPYHF=OFF -DENABLE_AIRSPYHF=ON
+}
+
+variant bladerf description {Enable bladeRF hardware} {
+    depends_lib-append      port:bladeRF
+    configure.args-replace  -DENABLE_BLADERF=OFF -DENABLE_BLADERF=ON
+}
+
+variant funcube description {Enable funcube hardware} {
+    configure.args-replace  -DENABLE_FUNCUBE=OFF -DENABLE_FUNCUBE=ON
+}
+
+variant hackrf description {Enable HackRF hardware} {
+    depends_lib-append      path:lib/libhackrf.dylib:hackrf
+    configure.args-replace  -DENABLE_HACKRF=OFF -DENABLE_HACKRF=ON
+}
+
+variant libiio description {Enable libiio support, like PlutoSDR hardware} {
+    depends_lib-append      port:libiio
+    configure.args-replace  -DENABLE_IIO=OFF -DENABLE_IIO=ON
+}
+
+variant limesuite description {Enable limesuite hardware} {
+    depends_lib-append      path:lib/libLimeSuite.dylib:limesuite
+    configure.args-replace  -DENABLE_LIMESUITE=OFF -DENABLE_LIMESUITE=ON
+}
+
+variant perseus description {Enable perseus sdr device} {
+    depends_lib-append      port:perseus-sdr
+    configure.args-replace  -DENABLE_PERSEUS=OFF -DENABLE_PERSEUS=ON
+}
+
+variant rtlsdr description {Enable rtl-sdr hardware} {
+    depends_lib-append      port:rtl-sdr
+    configure.args-replace  -DENABLE_RTLSDR=OFF -DENABLE_RTLSDR=ON
+}
+
+variant xtrx description {Enable xtrx hardware} {
+    configure.args-replace  -DENABLE_XTRX=OFF -DENABLE_XTRX=ON
+}
+
+variant server description {Enable server binary} {
+    configure.args-replace  -DBUILD_SERVER=OFF -DBUILD_SERVER=ON
+}
+
+variant gui description {Enable Gui} {
+    configure.args-replace  -DBUILD_GUI=OFF -DBUILD_GUI=ON
+
+    app.create yes
+    app.name SDRangel
+    app.executable sdrangel
+    app.icon cmake/cpack/sdrangel_icon.icns
+    app.retina yes
+}
+
+variant native description {Enable native cpu flags (recommended)} {
+    configure.args-replace  -DFORCE_SSE41=ON -DFORCE_SSE41=OFF
+}
+
+variant debug description {Enable debug messages} {
+    configure.args-append   \
+        -DDEBUG_OUTPUT=ON   \
+        -DCMAKE_BUILD_TYPE=Debug
+}
```

Comment:
Please sort the variants alphabetically.

Reply:
thank you!

Reply:
I'm not seeing that here.

Reply:
ops push.
variants are now organized in alphabetically order per type.
I mean, first the generic one than driver one to easily manage future update.